README.md

```diff
@@ -1,2 +1,5 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+# Artefacts
+## <a href="prd/Juntos - PRD.pdf">PRD</a>
```

Comment:
- A heading without any content seems out of place. I think this can be a better represented as a bullet point instead.
- Can we please use the the  markdown syntax for links.